.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+# Beta Feature Requirements
+
+This page lists the requirements for promoting a feature to beta. Promotion to beta must meet all requirements for promotion to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+
+**Design Doc:**
+
+**Alpha Checklist:**
+
+**Relevant Documentation:**
+
+--- 
+
+### Requirements: 
+
+**Design**
```

Comment:
isn't this duplicate with alpha reuirement

Reply:
I mean when this feature is migrated to beta without change

Reply:
Feature coverage and test plans aren't required for alpha. For alpha, we're just expecting an RFC with an idea of what you want to implement as well as what it it's needed for. Beta expands on that with a design doc including an expectation that you'll know how you want to implement it and have an idea of what sort of testing it might need. Some of the content overlaps between the RFC and design doc, but not all.

Reply:
@hzxuzhonghu does that make sense?